view.php

```diff
@@ -467,7 +467,18 @@
     } else if ($offlinequiz->showtutorial) {
         $url = new moodle_url($CFG->wwwroot . '/mod/offlinequiz/tutorial.php',
             array('id' => $cm->id));
-        redirect($url);
+        echo $OUTPUT->header();
```

Comment:
This code was retrieved from previous versions of offlinequiz. I liked it to much. Its easy to use and non-intrusive.
At fisrt I thought of forcing the student to do the tutorial, but then I throught that the first time it will be valueable but a pain in the ass after a full semester of exams. Moreover, for the student the description text and the USERID are the most important information pieces of the offlinequiz activity before the exam itselt.